bot/exts/filters/filter_lists.py

```diff
@@ -287,10 +293,58 @@ async def deny_sync(self, ctx: Context) -> None:
         """Syncs both allowlists and denylists with the API."""
         await self._sync_data(ctx)
 
+    @command(name="filter_report")
+    async def force_send_weekly_report(self, ctx: Context) -> None:
+        """Send a list of autobans added in the last 7 days to #mod-meta."""
+        await self.send_weekly_autoban_report()
+
+    @tasks.loop(time=datetime.time(hour=18))
+    async def weekly_autoban_report_task(self) -> None:
+        """Trigger autoban report to be sent if it is the desired day of the week (WEEKLY_REPORT_ISO_DAY)."""
+        if arrow.utcnow().isoweekday() != WEEKLY_REPORT_ISO_DAY:
+            return
+
+        await self.send_weekly_autoban_report()
+
+    async def send_weekly_autoban_report(self, channel: discord.abc.Messageable = None) -> None:
+        """Send a list of autobans added in the last 7 days to #mod-meta."""
+        seven_days_ago = arrow.utcnow().shift(days=-7)
```

Comment:
Given the actual time this is run could differ by up to 18 hours each week, I think it would be possible that any autobans added in (up to) the first 18 hours after a message *may* not appear in the next message.

We could make this `-8` days (or -7 and 18 hours), if we'd rather have repeats than missed ones. I don't mind either way. I don't think it's worth a more complex solution to 'fix' this though, as it seems pretty minor.

Reply:
Where do the 18 hours come from?

Reply:
https://github.com/python-discord/bot/pull/2267/files#diff-2d7d975ffaf01fcfb495567a4238c5d0e5fee119591ad115a83ea6ebf5471677R301 means that it will run at 6pm UTC every day. Rather than some 18 hour offset.

https://discordpy.readthedocs.io/en/latest/ext/tasks/index.html#discord.ext.tasks.loop

Reply:
I swear I got confused on the exact same thing on a different PR recently 🤦

Ignore what I said 😅